test/driver.js

```diff
@@ -468,12 +472,13 @@ var Driver = (function DriverClosure() {
               var textLayerContext = textLayerCanvas.getContext('2d');
               textLayerContext.clearRect(0, 0,
                 textLayerCanvas.width, textLayerCanvas.height);
+              var enhanceText = task.id.includes('enhance') ? true : false;
```

Comment:
I'd would probably be much better overall if a `enhance` parameter was added (similar to the `annotations` flag in PR #6780).